assets/js/pages/ChecksSelection/ChecksSelectionHeader.test.jsx

```diff
@@ -146,6 +149,40 @@ describe('ChecksSelectionHeader component', () => {
     }
   );
 
+  it('should forbid starting a check execution', async () => {
```

Comment:
This test looks wrong.
Here we should have the button disabled and the tooltip, as we do in the next test.
So we shouldn't be passing `all:all` in the userAbilities

Reply:
thanks for spotting, this morning I'm a little bit sleepy 😴